book/version_control.md

```diff
@@ -395,140 +336,102 @@ To handle a data breach, you should:
 
-Issues are a useful soundboard for requesting changes, but the implementation of changes are handled by Pull Requests (below).
+Once a PR has been reviewed and the reviewer is happy with the changes, the Conversation view can be used by internal developers to merge the PR changes onto the target branch.
 
 
-### Pull requests
+(forking)=
+### Forking
 
-A pull request lets you describe changes that you have made to a repo.
-The request is based on the difference between a target branch (usually `dev` or `master`) and a source branch, where you have implemented changes.
-The source branch can the in the same repository, or might be in a Fork (below) of the repository if you are not a member of the project.
+Forking a repository takes a complete copy of a project's current state, including its history and all existing branches and tags. Any changes made to a fork do not affect the code on the original repo, and *vice versa*.
 
-Pull requests create an interface for discussion and review of your changes.
-Once a project maintainer is happy with your changes, they can merge them onto the target branch.
-After merging, pull requests preserve a record of the changes and associated discussion.
+You might fork a repository when you want to:
+* Contribute to a project as an external collaborator
+* Make changes to a project for your own use, or to maintain a copy that is independent to the original
 
-You can label pull requests a draft to indicate they are still a work in progress.
-This prevents them from being merged prematurely.
-This can be useful when you would like to request advice or early feedback on the changes you are making.
+In the first case, lets consider that an issue describes a bug in a project's code. Looking at the code, you think that you know where the source of the bug is. You create a fork of the project and clone your copy of the project locally. Here you make commits that include changes to fix the bug and test that these changes work. Once the bug is fixed, you create a Pull Request on the original project and describe how your changes fix the bug. The source branch of this PR will be the branch that you committed to on your fork of the project, while the target will be the development or main branch of the original repo.
 
-Like issues, pull requests can have assignees that are working on them.
-You can also assign reviewers and tag (`@`) project collaborators as part of the discussion.
+In the second case, perhaps you want to reuse or adapt code from an existing project. If the changes that you wish to make to the code are not in line with the aim of the original project, then you might create a fork to contain these changes.
```

Comment:
Also a use case for this is when the original project become deprecated and not maintained and you want to keep it going.